### ISSUE TITLE
Proposal: ApplicationViewMode.CompactOverlay max size should be increased

### ISSUE THREAD

**Issue by fsclientdev** [microsoft/microsoft-ui-xaml#2656]
# Proposal: ApplicationViewMode.CompactOverlay max-size should be increased

## Summary
Right now with that application view mode user can't change size to value bigger than 500x500.

## Rationale
With Win32 API it is possible to create "top most" window without such limitation.
Edge on Chromium and Chrome also allow opening video in "picture in picture" mode with 700x520 maximum.

## Scope
| Capability | Priority |
| :---------- | :------- |
| Allow max size greater than 500x500 for CompactOverlay view mode | Must |
| Allow configuring min/max size with SetPreferredMinSize per view mode | Should |

## Notes
500x500 should remain as default max size, as it seems as preferred for most users.

## Open Questions
Not sure, if it is limited on WinRT API level on Windows.UI level.
Feel free to move this issue to ProjectReunion repository, if it should be there.

**Comment by ivanch88** [comment]
Yeah I agree with fsclientdev. 500x500 window is too small for the modern 2.5k or 4k monitors.

**Comment by fsclientdev** [comment]
It could be at least 1/4 of screen space.

**Comment by StephenLPeters** [comment]
@ryalanms and @ocalvo FYI

**Comment by SyavaK** [comment]
agree 500x500 for today is not enough

**Comment by tharmes42** [comment]
limit increase would be great - even more than 700x520 to allow video sources stay on top in full resolution in multi-monitor setups

**Comment by aquinn39** [comment]
Since this is likely something that needs to be changed in Windows itself rather than WinUI, also submitted on Feedback Hub here: https://aka.ms/AAgsfu1.
